src/shared/components/news-item.tsx

```diff
@@ -0,0 +1,37 @@
+import { Component } from "inferno";
+import { Helmet } from "inferno-helmet";
+import { i18n } from "../i18next";
+import { news_md } from "../translations/news";
+import { isBrowser, mdToHtml } from "../utils";
+
+const title = i18n.t("news");
+
+export class NewsItem extends Component<any, any> {
+  constructor(props: any, context: any) {
+    super(props, context);
+  }
+
+  componentDidMount() {
+    if (isBrowser()) {
+      window.scrollTo(0, 0);
+    }
+  }
+
+  get markdown(): string {
+    let title = decodeURIComponent(this.props.match.params.title);
+    return news_md.find(v => v.title == title).markdown;
+  }
```

Comment:
It looks a bit unusual to have spaces in the url, and it could potentially cause some problems for copy-paste. Maybe for the url we can replace spaces with underscore or dash, and convert everything to lowercase.